src/custom-element.js

```diff
@@ -189,6 +190,9 @@ export function applyLayout_(element) {
     assert(getLengthUnits(width) == getLengthUnits(height),
         'Length units should be the same for width and height: %s, %s',
         widthAttr, heightAttr);
+  } else {
+    assert(heightsAttr === null,
```

Comment:
I'm thinking it will also be convenient to modify code above a little to default layout when "heights" is specified. So, above code would change like this (can't add comment there):
```
- } else if (height && width && sizesAttr) {
+ } else if (height && width && (sizesAttr || heightsAttr)) {
```